modules/engines/engine-hdb/src/main/java/com/sap/xsk/hdb/ds/parser/hdbdd/XSKHdbddParser.java

```diff
@@ -84,15 +84,21 @@ public XSKDataStructureModel parse(XSKDataStructureParametersModel parametersMod
         this.symbolTable.clearSymbolsByFullName();
         this.symbolTable.clearEntityGraph();
         this.symbolTable.clearViewGraph();
-        throw new XSKDataStructuresException(e.getMessage());
+        throw new XSKDataStructuresException(e);
```

Comment:
Add exception message and information about the file that cannot be parsed - location and anything that will help to identify and debug the problem

Reply:
We do get such information, for instance: `"com.sap.xsk.parser.hdbdd.exception.CDSRuntimeException: Failed to parse file: gstr2/ITC_EXPIRED_CONFIG.hdbdd. Error at line: 6  - Before an entity element declaration only the 'key' keyword is allowed"`